data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/InternalActionsWML/test_role_types.cfg

```diff
@@ -1,5 +1,5 @@
 #####
-# API(s) being tested: [role],[recall]role=
+# API(s) being tested: [recall]role=
```

Comment:
Original was right.

Reply:
I'd describe this one as "requires [role] to work, but is testing that you can recall by the role".

Reply:
Because of the multiple units, it seems more a test of `[role]`s filter to me.

Reply:
That's what the second test in this file (test_role_2) is testing though, isn't it?